### PR TITLE
Colour straight lines with the palette

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Set the colour of straight lines with the palette

## Why?

So that these lines are format and scheme aware!

## Screenshots

| Before      | After      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/user-attachments/assets/6e9516a4-418c-4ff1-88ad-4995120feb1f
[after]: https://github.com/user-attachments/assets/b9c5299b-9527-473e-a1d2-c18221cb6638

### CODE REVIEW

**Review by arelra — APPROVED** [review]

File: dotcom-rendering/src/layouts/InteractiveLayout.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ import { canRenderAds } from '../lib/canRenderAds';
 import { getContributionsServiceUrl } from '../lib/contributions';
 import { decideTrail } from '../lib/decideTrail';
 import type { NavType } from '../model/extract-nav';
-import { palette as themePalette } from '../palette';
+import { palette, palette as themePalette } from '../palette';
```
Comment (domlander) [inline dotcom-rendering/src/layouts/InteractiveLayout.tsx:49]: I don't think we need it twice

**Review by domlander — APPROVED** [review]
One step closer!

**Comment by domlander** [comment]
There's a few chromatic diffs where a line above the author on labs pages has changed colour. I don't think these are expected.

**Comment by mxdvl** [comment]
> There's a few chromatic diffs where a line above the author on labs pages has changed colour. I don't think these are expected.

Indeed, fixed in https://github.com/guardian/dotcom-rendering/pull/11975/commits/ad22152bc3a90b5526867749c7ff2f12cfe010e8

**Pull request merged**